README.md

```diff
@@ -320,8 +320,10 @@ const MapContainer = ({dispatch}) => (
 ```
 Read more about [forward dispatching actions][forward-actions]
 
+### 4. Customize style.
+TBD
```

Comment:
Add here?

Reply:
Also add `theme` prop to 2. Mount Component section

Reply:
done